app/main.py

```diff
@@ -1,13 +1,31 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {"line": error["line_number"],
+            "column": error["column_number"],
+            "message": error["text"],
+            "name": error["code"],
+            "source": "flake8"}
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {"errors": [{"line": i["line_number"],
+                        "column": i["column_number"],
+                        "message": i["text"],
+                        "name": i["code"],
+                        "source": "flake8"}
+                       for i in errors],
```

Comment:
use more descriptive names for your loops, for example here `error` instead of `i`